Runar/Runar/Application/AppDelegate.swift

```diff
@@ -119,7 +125,7 @@ extension AppDelegate {
             runesData = _runesData
         }
         
-        MemoryStorage.GenerationRunes = GenerationRuneModel.create(fromData: runesData!)
+        MemoryStorage.generationRunes = GenerationRuneModel.create(fromData: runesData!)
```

Comment:
@GeraStupakov think, please, how to avoid force unwrap.

Reply:
methods loadRues() and loadWallpapers() completely rewritten in pull request #73